src/i18n/locales/en/translation.json

```diff
@@ -924,7 +951,7 @@
     "pasted": "Pasted!",
     "description": "Description",
     "description_full": "Full Description",
-    "success_message": "{{amount}}{{fiatAmount}} sent to {{destination}}",
+    "success_message": "{{amount}}{{fiatAmount}} are on their way to {{destination}}",
```

Comment:
@rolznz Did you change the default message that is being used everywhere on purpose? Should we add a separate success message for that?